irohad/torii/processor/impl/transaction_processor_impl.cpp

```diff
@@ -17,23 +17,25 @@ namespace iroha {
 
     using network::PeerCommunicationService;
 
-    static std::string composeErrorMessage(
-        const validation::TransactionError &tx_error) {
-      if (not tx_error.first.tx_passed_initial_validation) {
-        return (boost::format("Stateful validation error: transaction %s "
-                              "did not pass initial verification: "
-                              "checking '%s', error message '%s'")
+    namespace {
```

Comment:
Alternative is private static function, but as [this question](https://stackoverflow.com/questions/154469/unnamed-anonymous-namespaces-vs-static-functions) tells, it's better to use anonymous namespace to define transaction unit only methods.

Reply:
As I understand, it is related only to header files, isn't it?

Reply:
As I explored, this makes no sense in header files, because it will be duplicated while compiling in every file this header is included, but for .cpp it works perfectly